src/odemis/model/_metadata.py

```diff
@@ -61,11 +61,41 @@
 MD_LENS_RI = "Lens refractive index"  # float (ratio), refractive index
 MD_FILTER_NAME = "Filter name"  # str, product name of the light filter
 # TODO: might need to merge DWELL_TIME and EXP_TIME into INTEGRATION_TIME: the time each pixel receive energy
+
+# extended fibsem metadata
+MD_STAGE_POSITION = "Stage position"  # dict of str -> float, position of the stage (in m/rad) for each axis
+MD_SAMPLE_PRE_TILT = "Stage pre-tilt"  # (rad) pre-tilt of the stage / shuttle (tilt)
+
 # + SCANNED_DIMENSIONS: list of dimensions which were scanned instead of being acquired simultaneously
-MD_DWELL_TIME = "Pixel dwell time"  # s (float), time the electron beam spends per pixel
-MD_EBEAM_VOLTAGE = "Electron beam acceleration voltage"  # V (float), voltage used to accelerate the electron beam
-MD_EBEAM_CURRENT = "Electron beam emission current"  # A (float), emission current of the electron beam (typically, the probe current is a bit smaller and the spot diameter is linearly proportional)
-MD_EBEAM_SPOT_DIAM = "Electron beam spot diameter"  # m (float), approximate diameter of the electron beam spot (typically function of the current)
+MD_BEAM_SCAN_ROTATION = "Beam scan rotation"  # (rad) rotation of the beam scan
```

Comment:
Could you extend the comment to explain how this is different than `MD_ROTATION`